client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -174,15 +178,16 @@ const NewsletterSetup: Step = ( { navigation } ) => {
 						className="newsletter-setup__accent-color"
 						style={ {
 							backgroundImage: [
-								generateSwatchSVG( accentColor ),
-								...( accentColor ? [ getBackgroundImage( accentColor ) ] : [] ),
+								generateSwatchSVG( accentColor.color ),
+								getBackgroundImage( accentColor.color ),
```

Comment:
`getBackgroundImage` has to know about `default` value too because the green tick is showing before picking a color. Maybe pass the entire `accentColor` object and handle it inside the function for less noise (if default return '').

Reply:
Got it. That's not exactly what the PR is about but I will fix that too. 

> Maybe pass the entire accentColor object and handle it inside the function for less noise (if default return '').
I do not think this is the better option since the function is used for all other inputs too, i.e. Site Name and Brief Description.